apps/dashboard/src/components/subscribers/preferences/subscriber-preferences-toggles.tsx

```diff
@@ -0,0 +1,45 @@
+import { STEP_TYPE_TO_ICON } from '@/components/icons/utils';
+import { Step } from '@/components/primitives/step';
+import { Switch } from '@/components/primitives/switch';
+import { STEP_TYPE_TO_COLOR } from '@/utils/color';
+import { capitalize } from '@/utils/string';
+import { ChannelTypeEnum } from '@novu/shared';
+import { motion } from 'motion/react';
+
+const CHANNEL_LABELS_LOOKUP: Record<`${ChannelTypeEnum}`, string> = {
```

Comment:
Let's reuse [this](`https://github.com/novuhq/novu/blob/8414e7815e57c7ae4747660dd6a576fcf1a952f9/apps/dashboard/src/utils/constants.ts#L19`). Another duplication we can eliminate is [this](https://github.com/novuhq/novu/blob/8185062ca3215f4d3152316e148eafbc3126212d/apps/dashboard/src/components/workflow-editor/channel-preferences-form.tsx#L35).

Reply:
I was talking about the record map of channel and labels, this is several times in the codebase. If we add one more step we will have to change it in 20 places.